tests/test_main_sync_error_handling.py

```
@@ -0,0 +1,123 @@
+from tests.helpers import (
+    inline_to_thread,
+    make_patched_get_running_loop,
+)
+
+# =============================================================================
+# TestMatrixSyncLoopErrorHandling (converted from unittest.TestCase)
+# =============================================================================
+
+
+@pytest.mark.parametrize(
+    "raised_exc, logger_method, expected_substr",
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)